src/scenic/simulators/robosuite/model.scenic

```diff
@@ -0,0 +1,253 @@
+    'table_thickness': 0.05,
+    
+    # Robot properties
+    'robot_width': 0.2,
+    'robot_length': 0.2,
+    'robot_height': 0.5,
+    
+    # Control parameters
+    'control_gain': 3.0,
+    'control_limit': 0.3,
+    'position_tolerance': 0.02,
+    'height_tolerance': 0.02,
+    'gripper_open_steps': 20,
+    'gripper_close_steps': 30,
+    'max_control_steps': 100,
+    'max_lift_steps': 200
+}
+
+# Base classes
+class RoboSuiteObject(Object):
+    """Base class for all RoboSuite objects."""
+    density: DEFAULTS['density']
+    friction: DEFAULTS['friction']
+    solref: DEFAULTS['solref']
+    solimp: DEFAULTS['solimp']
+    shape: BoxShape()
+    allowCollisions: True
+
+# Table for arena setup
+class Table(RoboSuiteObject):
+    """Table in environment."""
+    isTable: True
+    width: DEFAULTS['table_width']
+    length: DEFAULTS['table_length']
+    height: DEFAULTS['table_thickness']
+    position: (0, 0, DEFAULTS['table_height'])
+
+# Base class for manipulable objects
+class ManipulationObject(RoboSuiteObject):
+    """Base class for objects that can be manipulated."""
+    color: DEFAULTS['default_color']
+
+# Primitive shape objects (matching RoboSuite's naming)
+class Box(ManipulationObject):
+    """Box object."""
+    objectType: "Box"
+    width: DEFAULTS['object_size']
+    length: DEFAULTS['object_size']
+    height: DEFAULTS['object_size']
+
+class Ball(ManipulationObject):
+    """Ball/sphere object."""
+    objectType: "Ball"
+    radius: DEFAULTS['object_size']
+    width: DEFAULTS['object_size'] * 2
+    length: DEFAULTS['object_size'] * 2
+    height: DEFAULTS['object_size'] * 2
+
+class Cylinder(ManipulationObject):
+    """Cylinder object."""
+    objectType: "Cylinder"
+    width: DEFAULTS['object_size'] * 2
+    length: DEFAULTS['object_size'] * 2
+    height: DEFAULTS['object_size'] * 4
+
+class Capsule(ManipulationObject):
+    """Capsule object."""
+    objectType: "Capsule"
+    width: DEFAULTS['object_size'] * 1.5
+    length: DEFAULTS['object_size'] * 1.5
+    height: DEFAULTS['object_size'] * 3
+
+# Complex objects (matching RoboSuite's naming)
+class Milk(ManipulationObject):
+    """Milk carton object."""
+    objectType: "Milk"
+
+class Cereal(ManipulationObject):
+    """Cereal box object."""
+    objectType: "Cereal"
+
+class Can(ManipulationObject):
+    """Can object."""
+    objectType: "Can"
+
+class Bread(ManipulationObject):
+    """Bread object."""
+    objectType: "Bread"
+
+class Bottle(ManipulationObject):
+    """Bottle object."""
+    objectType: "Bottle"
+
+class Hammer(ManipulationObject):
+    """Hammer object."""
+    objectType: "Hammer"
+
+class SquareNut(ManipulationObject):
+    """Square nut object."""
+    objectType: "SquareNut"
+
+class RoundNut(ManipulationObject):
+    """Round nut object."""
+    objectType: "RoundNut"
+
+# Robots (matching RoboSuite's naming)
+class Robot(RoboSuiteObject):
+    """Base robot class."""
+    robot_type: "Panda"
```

Comment:
For consistency with Scenic, please use camel case for these properties, e.g. `robotType`